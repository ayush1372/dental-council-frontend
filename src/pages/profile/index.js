import { useState } from 'react';

import AccountCircleIcon from '@mui/icons-material/AccountCircle';
import AppRegistrationIcon from '@mui/icons-material/AppRegistration';
import BadgeIcon from '@mui/icons-material/Badge';
import CreditCardOffIcon from '@mui/icons-material/CreditCardOff';
import DashboardIcon from '@mui/icons-material/Dashboard';
import { Grid } from '@mui/material';
import { useDispatch, useSelector } from 'react-redux';

import TrackStatus from '../../shared/track-status/index';
import { changeUserActiveTab } from '../../store/reducers/ui-reducers';
import NewDoctorRegistration from '../profile/new-doctor-registration/new-doctor-registration';
import SuspendLicenseVoluntaryRetirement from '../suspend-license-voluntary-retirement';
import UserProfile from '../user-profile';
import CollegeDean from './college-dean/college-dean';
import CollegeMyProfile from './college-my-profile/college-my-profile';
import CollegeRegistrar from './college-registrar/college-registrar';
import Dashboard from './components/dashboard-cards/dashboard-cards';
import MiniDrawer from './components/profile-sidebar/profile-sidebar';
import MyProfile from './smc-nmc-profile/my-profile';

export function Profile() {
  const dispatch = useDispatch();
  const loggedInUserType = useSelector((state) => state.login.loggedInUserType);
  const { userActiveTab } = useSelector((state) => state.ui);

  const doctorTabs = [
    {
      option: 0,
      name: 'My Profile',
      tabName: 'my-profile',
      icon: <AccountCircleIcon />,
      element: <UserProfile tabName={userActiveTab} />,
    },
    {
      option: 1,
      name: 'Suspend License',
      tabName: 'suspend-license',
      icon: <CreditCardOffIcon />,
      element: <SuspendLicenseVoluntaryRetirement tabName={userActiveTab} />,
    },
    {
      option: 2,
      name: 'Voluntary Retirement',
      tabName: 'voluntary-retirement',
      icon: <CreditCardOffIcon />,
      element: <SuspendLicenseVoluntaryRetirement tabName={userActiveTab} />,
    },
  ];

  const smcTabs = [
    {
      option: 0,
      name: 'Dashboard',
      tabName: 'dashboard',
      icon: <DashboardIcon />,
      element: <Dashboard tabName={userActiveTab} />,
    },
    {
      option: 1,
      name: 'My Profile',
      tabName: 'my-profile',
      icon: <AccountCircleIcon />,
      element: <MyProfile userType={'SMC'} />,
    },
    {
      option: 2,
      name: 'Track Status',
      tabName: 'track-status',
      icon: <BadgeIcon />,
      element: <TrackStatus tabName={userActiveTab} />,
    },
    {
      option: 3,
      name: 'New Doctor Registration',
      tabName: 'New-doctor-registration',
      icon: <AppRegistrationIcon />,
      element: <NewDoctorRegistration userType={'SMC'} />,
    },
  ];

  const nmcTabs = [
    {
      option: 0,
      name: 'Dashboard',
      tabName: 'dashboard',
      icon: <DashboardIcon />,
      element: <Dashboard tabName={userActiveTab} />,
    },
    {
      option: 1,
      name: 'My Profile',
      tabName: 'my-profile',
      icon: <AccountCircleIcon />,
      element: <MyProfile userType={'NMC'} />,
    },
    {
      option: 2,
      name: 'Track Status',
      tabName: 'track-status',
      icon: <BadgeIcon />,
      element: <TrackStatus tabName={userActiveTab} />,
    },
  ];

  const colgTabs = [
    {
      option: 0,
      name: 'Dashboard',
      tabName: 'dashboard',
      icon: <DashboardIcon />,
      element: <Dashboard tabName={userActiveTab} />,
    },
    {
      option: 1,
      name: 'My Profile',
      tabName: 'my-profile',
      icon: <AccountCircleIcon />,
      element: <CollegeMyProfile />,
    },
    {
      option: 2,
      name: 'Track Status',
      tabName: 'track-status',
      icon: <BadgeIcon />,
      element: <TrackStatus tabName={userActiveTab} />,
    },
    {
      option: 3,
      name: 'College Registrar',
      tabName: 'college-registrar',
      icon: <AppRegistrationIcon />,
      element: <CollegeRegistrar />,
    },
    {
      option: 4,
      name: 'College Dean',
      tabName: 'college-dean',
      icon: <AppRegistrationIcon />,
      element: <CollegeDean />,
    },
  ];

  const [isActiveTab, setIsActiveTab] = useState(
    loggedInUserType === 'Doctor'
      ? doctorTabs[0].tabName
      : loggedInUserType === 'College' || loggedInUserType === 'SMC' || loggedInUserType === 'NMC'
      ? colgTabs[0].tabName
      : ''
  );

  const setActiveTab = (activeTab) => {
    setIsActiveTab(activeTab);
    dispatch(changeUserActiveTab(activeTab));
  };

  return (
    <Grid container>
      <Grid item xs={12}>
        <MiniDrawer
          DrawerOptions={
            loggedInUserType === 'Doctor'
              ? doctorTabs
              : loggedInUserType === 'NMC'
              ? nmcTabs
              : loggedInUserType === 'SMC'
              ? smcTabs
              : loggedInUserType === 'College'
              ? colgTabs
              : ''
          }
          handleSwitch={setActiveTab}
          ActiveOption={isActiveTab}
        />
      </Grid>
    </Grid>
  );
}

export default Profile;
