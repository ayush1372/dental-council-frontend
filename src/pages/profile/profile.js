// import styles from './profile-new.module.scss';

import { useState } from 'react';

import CloseIcon from '@mui/icons-material/Close';
import { Alert, Box, Collapse, Container, IconButton } from '@mui/material';

import { AbhaNumberCard } from './components/abha-number-card/abha-number-card';
// import { LoginSetPassword } from './components/login-set-password/login-set-password';

// import { verboseLog } from '../../config/debug';
// import  { PersistentDrawerLeft, ProfileDrawer } from './components/profile-drawer/profile-drawer';

export function Profile() {
  const [open, setOpen] = useState(true);
  // const [openDrawer, setOpenDrawer] = useState(false);

  // verboseLog('Drawer State',openDrawer)
  return (
    <Container maxWidth="md" sx={{ margin: '20px auto 20px', padding: '10px' }}>
      <Box>
        <Box>
          <Collapse in={open}>
            <Alert
              action={
                <IconButton
                  aria-label="close"
                  size="small"
                  onClick={() => {
                    setOpen(false);
                  }}
                >
                  <CloseIcon color="success" fontSize="inherit" />
                </IconButton>
              }
              sx={{ m: 2, marginLeft: '0px', borderRadius: '5px' }}
            >
              Your Mobile Number Is successfully updated.
            </Alert>
          </Collapse>
        </Box>
        {/* <Typography variant="h2" textAlign="center" component="span">
          Welcome to Your Profile
        </Typography> */}
        {/* <LoginSetPassword /> */}
        <AbhaNumberCard />
        {/* <Button 
          variant='contained'
          color='primary'
          onClick={() => setOpenDrawer(true)}
        >
          Open
        </Button>
        <Button 
          variant='contained'
          color='secondary'
          onClick={() => setOpenDrawer(false)}
        >
          Close
        </Button> */}
      </Box>
      {/* <ProfileDrawer />
      {/* <ResponsiveDrawer /> */}
      {/* {openDrawer && <PersistentDrawerLeft/>} */}
    </Container>
  );
}

export default Profile;
