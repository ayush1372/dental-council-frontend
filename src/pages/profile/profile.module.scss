.plusImage {
  position: absolute;
  bottom: 0;
  left: 0;
}
.profilePage {
  background-color: #e5eaef58;
  .profilePageContainer {
    padding: 40px;
    margin: auto;
  }
}

.profileContainer {
  margin-top: 10px;
  width: 20%;
  background-color: #ffffff;
  border-radius: 8px;
  min-height: 550px;
  .profileImgBackground {
    background: linear-gradient(to bottom, #e9ecf3 0%, #e9ecf3 50%, #ffffff 20%, #ffffff 100%);
  }
}
.tabDetailsContainer {
  width: 78%;
  // background-color: #ffffff;
  border-radius: 8px;
}
