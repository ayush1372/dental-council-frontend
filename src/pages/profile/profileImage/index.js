import { Grid, Typography } from '@mui/material';

import avtarImg from '../../../assets/images/user.png';

import styles from './profile-image.module.scss';

export default function ProfileImage(props) {
  return (
    <Grid container mt={0} spacing={1} className={styles.profileImageDetailsContainer}>
      <Grid item xs={12} display="flex" justifyContent="center" className={styles.imageContainer}>
        <img alt="avtarImg" className={styles.profileImage} src={avtarImg} />
      </Grid>
      <Grid textAlign="center" item xs={12}>
        <Typography className={styles.profileImageName}>
          {props.userType === 'professional' ? 'Dr.' : ''}
          {` ${props.name}`}
        </Typography>
      </Grid>
    </Grid>
  );
}
