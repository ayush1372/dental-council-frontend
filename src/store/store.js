/* eslint-disable simple-import-sort/imports */
import { configureStore } from '@reduxjs/toolkit';
import logger from 'redux-logger';
import thunk from 'redux-thunk';

import loginActivity from './reducers/common-reducers';
import contactReducer from './reducers/contact-reducer';
import appFontSizeActivity from './reducers/app-font-size';
import RegisterDrivingLicenceReducer from './reducers/register-driving-licence-reducers';
import captchaReducer from './reducers/captcha-reducers';

const store = configureStore({
  reducer: {
    login: loginActivity,
    getUserList: contactReducer,
    appFontSize: appFontSizeActivity,
    getRegisterDrivingLicence: RegisterDrivingLicenceReducer,
    getCaptchaData: captchaReducer,
  },
  middleware: (getDefaultMiddleware) => getDefaultMiddleware(thunk).concat(logger),
  devTools: true,
});

export default store;
