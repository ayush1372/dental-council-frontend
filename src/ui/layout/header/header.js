import 'react-toastify/dist/ReactToastify.css';

import { Box } from '@mui/material';

import { Navbar } from '../../core/navigation/navigation';
import { LogoWrapper } from './components/logo-wrapper/logo-wrapper';
import { TopBar } from './components/top-bar/top-bar';

// import { useDispatch, useSelector } from 'react-redux';
// import { Slide, toast, ToastContainer } from 'react-toastify';
// import { getNavMeta } from '../../../constants/navigation-meta';
// import { login, logout } from '../../../store/reducers/common-reducers';
import styles from './header.module.scss';

export const Header = () => {
  // const navigate = useNavigate();
  // const loggedIn = useSelector((state) => state.login.isloggedIn);
  // const navMeta = getNavMeta(loggedIn);
  // const time = useSelector((state) => state.login.timer);
  // const dispatch = useDispatch();
  // const [timeOutId, setTimeOutId] = useState(0);

  // const setLoginTimeOut = () =>
  //   setTimeout(() => {
  //     stopTimer();
  //   }, 0.5 * 60000);

  // function redirectHome() {
  //   // document.getElementById('HomeTab').click();
  //   navigate('/');
  // }

  // function startTimer() {
  //   dispatch(login());
  //   setTimeOutId(setLoginTimeOut());
  //   successToast('Login Successful', 'login', 'success');
  // }

  // function stopTimer() {
  //   if (timeOutId) {
  //     clearTimeout(timeOutId);
  //     setTimeOutId(0);
  //   }
  //   dispatch(logout());
  //   successToast('Logout Successful', 'logout', 'error');
  //   redirectHome();
  // }

  // const activeToggle = ({ isActive }) => ({
  //   ...(isActive ? { textDecoration: 'underline', color: 'red' } : null),
  // });

  return (
    <Box>
      <TopBar />
      <LogoWrapper />
      <Navbar />
      <div className={styles.main}>
        {/* <div>
          {loggedIn ? (
            <div className={styles.loginDiv}>
              <Button
                variant="contained"
                color="secondary"
                data-testid="logoutbtn"
                onClick={stopTimer}
              >
                {t('Logout')}
              </Button>
              <p data-testid="timer">{time}</p>
            </div>
          ) : (
            <div className={styles.loginDiv}>
              <Button
                variant="contained"
                color="primary"
                data-testid="loginbtn"
                onClick={startTimer}
              >
                {t('Login')}
              </Button>
              <Button variant="contained" color="secondary" onClick={() => navigate('/signup')}>
                {t('Sign Up')}
              </Button>
            </div>
          )}
        </div> */}
        {/* <div>
          <Button variant="contained" color="primary" onClick={() => navigate('/customform')}>
            {'Custom Form'}
          </Button>
        </div> */}

        {/* <ToastContainer></ToastContainer> */}
      </div>
    </Box>
  );
};

// const successToast = (message, operation, toastType) => {
//   toast(message, {
//     toastId: `${operation}-success-toast`,
//     position: 'bottom-right',
//     transition: Slide,
//     type: toastType,
//     autoClose: 2000,
//     hideProgressBar: true,
//     closeButton: false,
//     closeOnClick: false,
//     draggable: false,
//     style: {
//       borderRadius: '5px',
//       textAlign: 'center',
//       color: '#0a203e',
//       fontSize: '16px',
//       letterSpacing: '1.2px',
//       fontWeight: '500',
//     },
//   });
// };
