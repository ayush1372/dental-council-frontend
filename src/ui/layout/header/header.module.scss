.main {
  display: flex;
  align-items: center;
  padding: 0px 0px;
}
.loginDiv {
  display: inline-flex;
  margin: 0px 30px;
}
