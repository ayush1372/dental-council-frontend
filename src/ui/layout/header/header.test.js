import { render, screen } from '@testing-library/react';
import userEvent from '@testing-library/user-event';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import { ToastContainer } from 'react-toastify';

import { Profile } from '../../../pages/profile/profile';
import store from '../../../store/store';
import { Header } from './header';

beforeEach(() => {
  render(
    <Router>
      <Provider store={store}>
        <Header />
      </Provider>
    </Router>
  );
});

describe('login', () => {
  test('renders "Login" text on button initially', () => {
    const InidisplayedText = screen.getByText('Login');
    expect(InidisplayedText).toBeInTheDocument();
  });

  test('Header component snapshot', () => {
    const { container } = render(
      <Router>
        {' '}
        <Provider store={store}>
          <Header />
        </Provider>
      </Router>
    );
    expect(container).toMatchSnapshot();
  });

  test('Button Text changes to Logout on Login Click', () => {
    const buttonElement = screen.getByTestId('loginbtn');
    userEvent.click(buttonElement);
    const LoginText = screen.getByText('Logout');
    expect(LoginText).toBeInTheDocument();
  });

  test('Login Successful message is displayed', async () => {
    const buttonElement = screen.getByTestId('logoutbtn');
    userEvent.click(buttonElement);
    userEvent.click(buttonElement);
    render(<ToastContainer></ToastContainer>);
    expect(await screen.findByText('Login Successful')).toBeInTheDocument();
  });

  test('User Profile is displayed', () => {
    const buttonElement = screen.getByTestId('logoutbtn');
    userEvent.click(buttonElement);
    userEvent.click(buttonElement);
    render(<Profile></Profile>);
    const UserProfileText = screen.getByText('User Profile');
    expect(UserProfileText).toBeInTheDocument();
  });

  test('Timer starts on Login Click', () => {
    const buttonElement = screen.getByTestId('logoutbtn');
    userEvent.click(buttonElement);
    userEvent.click(buttonElement);
    const TimerElement = screen.getByTestId('timer');
    expect(TimerElement).not.toBeNull();
  });

  test('Login Button Text changes to Logout on Login Click', () => {
    const buttonElement = screen.getByTestId('logoutbtn');
    userEvent.click(buttonElement);
    userEvent.click(buttonElement);
    const LoginText = screen.getByText('Logout');
    expect(LoginText).toBeInTheDocument();
  });

  test('Login Successful message will be displayed', async () => {
    const buttonElement = screen.getByTestId('logoutbtn');
    userEvent.click(buttonElement);
    userEvent.click(buttonElement);
    render(<ToastContainer></ToastContainer>);
    expect(await screen.findByText('Login Successful')).toBeInTheDocument();
  });

  test('User Profile Tab is displayed', () => {
    const buttonElement = screen.getByTestId('logoutbtn');
    userEvent.click(buttonElement);
    userEvent.click(buttonElement);
    render(<Profile></Profile>);
    const UserProfileText = screen.getByText('User Profile');
    expect(UserProfileText).toBeInTheDocument();
  });

  test('Login Timer starts on Login Click', () => {
    const buttonElement = screen.getByTestId('logoutbtn');
    userEvent.click(buttonElement);
    userEvent.click(buttonElement);
    const TimerElement = screen.getByTestId('timer');
    expect(TimerElement).not.toBeNull();
  });
});

describe('logout', () => {
  test('Logout Successful message will be displayed', async () => {
    const buttonElement = screen.getByTestId('logoutbtn');
    userEvent.click(buttonElement);
    userEvent.click(buttonElement);
    render(<ToastContainer></ToastContainer>);
    expect(await screen.findByText('Logout Successful')).toBeInTheDocument();
  });

  test('renders "Login" text on Logout button when loggedout', () => {
    const buttonElement = screen.getByTestId('logoutbtn');
    userEvent.click(buttonElement);
    const LogoutText = screen.getByText('Login');
    expect(LogoutText).toBeInTheDocument();
  });

  test('Logout Successful  message is displayed', async () => {
    const buttonElement = screen.getByTestId('logoutbtn');
    userEvent.click(buttonElement);
    render(<ToastContainer></ToastContainer>);
    expect(await screen.findByText('Logout Successful')).toBeInTheDocument();
  });
});
