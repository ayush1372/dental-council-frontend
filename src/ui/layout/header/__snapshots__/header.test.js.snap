// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`login Header component snapshot 1`] = `
<div>
   
  <div
    class="MuiBox-root css-0"
  >
    <div
      class="topBarWrapper MuiBox-root css-0"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      >
        <div
          class="MuiGrid-root MuiGrid-container css-8qwpet-MuiGrid-root"
        >
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-7 MuiGrid-grid-md-6 css-1w65gxq-MuiGrid-root"
          >
            <svg
              class=""
              fill="currentColor"
              height="14px"
              width="14px"
            >
              call_black_48dp.svg
            </svg>
            <p
              class="MuiTypography-root MuiTypography-body2 css-4002zd-MuiTypography-root"
            >
              Our Toll Free Number
              : 14477
            </p>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-5 MuiGrid-grid-md-6 css-1uvoj4m-MuiGrid-root"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium topBarIcon css-1e6y48t-MuiButtonBase-root-MuiButton-root"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-endIcon MuiButton-iconSizeMedium css-9tj150-MuiButton-endIcon"
              >
                <svg
                  class=""
                  fill="currentColor"
                  height="20px"
                  width="20px"
                >
                  visibility_black_48dp.svg
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium topBarIcon css-1e6y48t-MuiButtonBase-root-MuiButton-root"
              tabindex="0"
              type="button"
            >
              <div>
                <button
                  aria-controls="menu-appbar"
                  aria-haspopup="true"
                  aria-label="language-options"
                  class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-sizeMedium fontZoomSmall css-mgcv3e-MuiButtonBase-root-MuiIconButton-root"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="fontSmall"
                  >
                    A
                  </span>
                   
                </button>
              </div>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium topBarIcon css-1e6y48t-MuiButtonBase-root-MuiButton-root"
              tabindex="0"
              type="button"
            >
              <div>
                <button
                  aria-controls="menu-appbar"
                  aria-haspopup="true"
                  aria-label="fontSize-options"
                  class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-sizeMedium fontZoomMedium css-mgcv3e-MuiButtonBase-root-MuiIconButton-root"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="fontMedium"
                  >
                    A
                  </span>
                   
                </button>
              </div>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium topBarIcon css-1e6y48t-MuiButtonBase-root-MuiButton-root"
              tabindex="0"
              type="button"
            >
              <div>
                <button
                  aria-controls="menu-appbar"
                  aria-haspopup="true"
                  aria-label="fontSize-options"
                  class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-sizeMedium fontZoomLarge css-mgcv3e-MuiButtonBase-root-MuiIconButton-root"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="fontLarge"
                  >
                    A
                  </span>
                   
                </button>
              </div>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium topBarIcon css-1e6y48t-MuiButtonBase-root-MuiButton-root"
              tabindex="0"
              type="button"
            >
              <button
                aria-controls="menu-appbar"
                aria-haspopup="true"
                aria-label="language-options"
                class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-sizeMedium languageIcon css-mgcv3e-MuiButtonBase-root-MuiIconButton-root"
                tabindex="0"
                type="button"
              >
                <svg
                  class=""
                  fill="currentColor"
                  height="20px"
                  width="20px"
                >
                  ico-language.svg
                </svg>
              </button>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium topBarIcon css-1e6y48t-MuiButtonBase-root-MuiButton-root"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-endIcon MuiButton-iconSizeMedium css-9tj150-MuiButton-endIcon"
              >
                <svg
                  class=""
                  fill="currentColor"
                  height="20px"
                  width="20px"
                >
                  help_outline_black_24dp.svg
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium topBarIcon css-1e6y48t-MuiButtonBase-root-MuiButton-root"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-endIcon MuiButton-iconSizeMedium css-9tj150-MuiButton-endIcon"
              >
                <svg
                  class=""
                  fill="currentColor"
                  height="20px"
                  width="20px"
                >
                  ico-notification.svg
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-sizeMedium css-whjmaz-MuiButtonBase-root-MuiIconButton-root"
              tabindex="0"
              type="button"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="Brightness4Icon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 8.69V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12 20 8.69zM12 18c-.89 0-1.74-.2-2.5-.55C11.56 16.5 13 14.42 13 12s-1.44-4.5-3.5-5.45C10.26 6.2 11.11 6 12 6c3.31 0 6 2.69 6 6s-2.69 6-6 6z"
                />
              </svg>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
    >
      <div
        class="MuiGrid-root MuiGrid-container css-8qwpet-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6 css-1kbptzk-MuiGrid-root"
        >
          <div
            class="logoWrapper MuiBox-root css-0"
          >
            <a
              class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1ps4owl-MuiTypography-root-MuiLink-root"
              href="/"
            >
              <img
                alt="NHA logo"
                class="logoImage"
                src="nha-logo.png"
              />
            </a>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6 css-k22mtw-MuiGrid-root"
        >
          <div
            class="MuiGrid-root MuiGrid-container css-gbj4yw-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-9 css-4um21y-MuiGrid-root"
            >
              <div
                class="MuiBox-root css-1djk842"
              >
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeMedium MuiButton-containedSizeMedium css-zcbmsk-MuiButtonBase-root-MuiButton-root"
                  tabindex="0"
                  type="button"
                >
                  Create your ABHA Number
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
              </div>
              <div
                class="authBlock MuiBox-root css-unpvr7"
              >
                <h6
                  class="MuiTypography-root MuiTypography-subtitle2 css-23bz3x-MuiTypography-root"
                >
                  Already have a ABHA Number?
                </h6>
                <a
                  class="MuiTypography-root MuiTypography-body1 MuiLink-root MuiLink-underlineAlways css-1pe1wbp-MuiTypography-root-MuiLink-root"
                  href="/login"
                >
                  login
                </a>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 css-1o7apob-MuiGrid-root"
            >
              <img
                alt="Abha number"
                src="abha-number.svg"
                width="96px"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiBox-root css-0"
    >
      <div
        class="menuBar MuiBox-root css-0"
      >
        <span />
        <span />
        <span />
      </div>
      <div
        class="navBarWrapper MuiBox-root css-0"
      >
        <div
          class="MuiContainer-root MuiContainer-maxWidthLg MuiContainer-disableGutters menuBlock css-1p4tas-MuiContainer-root"
        >
          <div
            class="menuWrapper MuiBox-root css-0"
          >
            <button
              aria-haspopup="true"
              class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium menuButton css-sghohy-MuiButtonBase-root-MuiButton-root"
              id="basic-button"
              tabindex="0"
              type="button"
            >
              Home
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
            <button
              aria-haspopup="true"
              class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium menuButton css-sghohy-MuiButtonBase-root-MuiButton-root"
              id="basic-button"
              tabindex="0"
              type="button"
            >
              About ABDM
              <span
                class="MuiButton-endIcon MuiButton-iconSizeMedium css-9tj150-MuiButton-endIcon"
              >
                <svg
                  class=""
                  fill="currentColor"
                  height="24px"
                  width="24px"
                >
                  keyboard_arrow_down_black_48dp.svg
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
            <button
              aria-haspopup="true"
              class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium menuButton css-sghohy-MuiButtonBase-root-MuiButton-root"
              id="basic-button"
              tabindex="0"
              type="button"
            >
              FAQ
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
            <button
              aria-haspopup="true"
              class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium menuButton css-sghohy-MuiButtonBase-root-MuiButton-root"
              id="basic-button"
              tabindex="0"
              type="button"
            >
              Contact Us
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
        <div
          class="MuiBox-root css-1srvgqb"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedSecondary MuiButton-sizeSmall MuiButton-containedSizeSmall css-ryyx5b-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="button"
          >
            Facility Login
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
      </div>
    </div>
    <div
      class="main"
    />
  </div>
</div>
`;
